demisto_sdk/commands/common/clients/xsoar/xsoar_api_client.py

```diff
@@ -618,6 +620,42 @@ def get_integration_instance(self, instance_name: str):
 
         raise ValueError(f"Could not find instance for instance name '{instance_name}'")
 
+    def disable_integration_instance(self, instance_name):
+        return self.update_integration_instance_state(False, instance_name)
+
+    def enable_integration_instance(self, instance_name):
+        return self.update_integration_instance_state(True, instance_name)
+
+    def update_integration_instance_state(self, enable: bool, instance_name: str):
+
+        # it will throw an error if the instance does not exist
+        instance = self.get_integration_instance(instance_name)
```

Comment:
consider try except if it throws an error

Reply:
its already handled in the get_integration_instance method itself